tests/lib/tools/snaps-state

```diff
@@ -59,6 +62,92 @@ install_local_as() {
     install_local "$snap" --name "$name" "$@"
 }
 
+is_classic_confinement_supported() {
+    snap debug sandbox-features --required=confinement-options:classic
+}
+
+# The function will re-pack either core or snpad snap using the assets
+# from the snapd deb installed on the system
+repack_snapd_deb_into_snap() {
+    local SNAP_NAME="$1"
+
+    if  [ "$SNAP_NAME" = "core" ]; then
```

Comment:
A switch statement would be more natural here

Reply:
sure